src/confluent_kafka/schema_registry/json_schema.py

```diff
@@ -172,25 +169,23 @@ def __init__(self, schema_str, schema_registry_client, to_dict=None,
 
     def __call__(self, obj, ctx):
         """
-        Serializes an object to the Confluent Schema Registry's JSON binary
-        format.
+        Serializes an object to JSON, prepending it with Confluent Schema Registry
+        framing.
 
         Args:
-            obj (object): object instance to serialize.
+            obj (object): The object instance to serialize.
 
             ctx (SerializationContext): Metadata pertaining to the serialization
                 operation.
 
-        Note:
-            None objects are represented as Kafka Null.
-
         Raises:
             SerializerError if any error occurs serializing obj
 
         Returns:
-            bytes: Confluent Schema Registry formatted JSON bytes
-
+            bytes: None if obj is None, else a byte array containing the JSON
```

Comment:
I think it is worthwhile to explain why it is okay to return None and what happens in the producer (namely the Kafka Null tombstone).